ansys/mapdl/core/launcher.py

```diff
@@ -329,6 +348,12 @@ def launch_grpc(exec_file='', jobname='file', nproc=2, ram=None,
         port += 1
     pymapdl._LOCAL_PORTS.append(port)
 
+    # setting ip for the grpc server
+    _using_custom_grpc_ip = False 
```

Comment:
There's no reason to use a leading underscore to make the parameter "protected" since these parameters won't be exposed to the user.  Fix here and all usages of it.

Reply:
Fixed in the new commit.